src/IRrecv.cpp

```diff
@@ -1014,6 +1014,10 @@ bool IRrecv::decode(decode_results *results, irparams_t *save,
     DPRINTLN("Attempting XMP decode");
     if (decodeXmp(results, offset, kXmpBits)) return true;
 #endif  // DECODE_XMP
+#if DECODE_KELON
+    DPRINTLN("Attempting Kelon decode");
+    if (decodeKelon(results, offset, kKelonBits)) return true;
+#endif // DECODE_KELON
```

Comment:
src/IRrecv.cpp:1020:  At least two spaces is best between code and comments  [whitespace/comments] [2]